incubator/react-native-error-trace-decorator/src/utils.ts

```diff
@@ -0,0 +1,66 @@
+import * as fse from "fs-extra";
+import * as ChildProcess from "child_process";
+import { error } from "@rnx-kit/console";
+import type { IConfigFile } from "./types";
+
+/**
+ * Takes a buffer, joins it with a newline and symbolicates it with the provided sourcemap
+ *
+ * @param buffer    a string array representing the buffer
+ * @param sourcemap sourcemap that must be used to symbolicate the passed buffer
+ */
+export const symbolicateBuffer = (buffer: string[], sourcemap: string) => {
```

Comment:
For legibility reasons, we should use `function` instead of arrow-functions:

```suggestion
export function symbolicateBuffer(buffer: string[], sourcemap: string): void {
```

This applies to all files.

Reply:
Replaced with "function"